controllers/contacts-controller.js

```diff
@@ -18,14 +18,24 @@ const getContactById = async (req, res) => {
 
 const addContact = async (req, res) => {
   const { body } = req;
-  const result = await contactsService.addContact(body);
+  const result = await Contact.create(body);
   res.status(201).json(result);
 };
 
 const updateContact = async (req, res) => {
   const { id } = req.params;
   const { body } = req;
-  const result = await contactsService.updateContact(id, body);
+  const result = await Contact.findByIdAndUpdate(id, body, { new: true });
+  if (!result) {
+    throw HttpError(404, `Not found`);
+  }
+  res.json(result);
+};
+
+const updateStatusContact = async (req, res) => {
+  const { id } = req.params;
+  const { body } = req;
```

Comment:
Warto doprecyzować zawsze co przychodzi, bo tu np favorite co nie?

Reply:
Osobiście bym to nazwał updateFavoriteStatus, bo updateStatusContact może oznaczać cokolwiek, ale taka nazwa była podana w wytycznych zadania więc tego się trzymałem.